.github/workflows/ci.yml

```diff
@@ -0,0 +1,225 @@
+name: CI
+
+on: ["push", "pull_request"]
+
+env:
+  TRAVIS_MAKE: make -f .travis.mk
+
+jobs:
+  luacheck:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+    container:
+      image: ghcr.io/avtikhon/tarantool/testing/debian-stretch:latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: test
+        run: ${TRAVIS_MAKE} test_debian_luacheck
+
+  release:
+    needs: [luacheck]
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+    container:
+      image: ghcr.io/avtikhon/tarantool/testing/debian-stretch:latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: test
+        run: ${TRAVIS_MAKE} test_debian_no_deps
+      - name: artifacts
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: release
+          retention-days: 1000000
+          path: test/var/artifacts
+
+  debug:
+    needs: [luacheck]
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+    container:
+      image: ghcr.io/avtikhon/tarantool/testing/debian-stretch:latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: test
+        run: ${TRAVIS_MAKE} test_coverage_debian_no_deps
```

Comment:
I don't see a comment from Coveralls in this PR. Although with Travis CI there was a comment with measured code coverage value, see for example [1]. It looks like degradation in comparison to Travis CI pipeline.

1. https://github.com/tarantool/tarantool/pull/5618#issuecomment-743978691

Reply:
Missing coveralls problems will be fixed in separate issue #5644. We have degradation here (comparing to travis flow), but it's not blocking.